public/js/buffalo/interface.js

```diff
@@ -0,0 +1,72 @@
+const main = document.getElementById('specs')
```

Comment:
Nice! Looks like you had fun polishing your test framework :)

Reply:
Thank you Simo 🙂  appreciate the helpful feedback, looking forward to applying your suggestions this week